corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -320,28 +377,64 @@ hqDefine("geospatial/js/case_grouping_map",[
                 let caseGroup = caseGroupsInstance.getGroupByID(caseGroupID);
                 color = caseGroup.color;
                 const marker = new mapboxgl.Marker({ color: color, draggable: false });  // eslint-disable-line no-undef
-                marker.setLngLat([caseItem.coordinates.lng, caseItem.coordinates.lat]);
+
+                const coordinates = [caseItem.coordinates.lng, caseItem.coordinates.lat];
+                marker.setLngLat(coordinates);
 
                 // Add the marker to the map
                 marker.addTo(map);
                 mapMarkers.push(marker);
+
+                let popupDiv = document.createElement("div");
+                popupDiv.setAttribute("data-bind", "template: 'select-case'");
+
+                let popup = new mapboxgl.Popup({ offset: 25, anchor: "bottom" })  // eslint-disable-line no-undef
+                    .setLngLat(coordinates)
+                    .setDOMContent(popupDiv);
+
+                marker.setPopup(popup);
+
+                const markerDiv = marker.getElement();
+                // Show popup on hover
+                markerDiv.addEventListener('mouseenter', () => marker.togglePopup());
```

Comment:
Nit: Since we aren't passing any params, we can shorten this to `markerDiv.addEventListener('mouseenter', marker.togglePopup)`

Reply:
Changing it to the suggestion will result in the function executing on load instead of executing the function on mouse enter.

Reply:
That's interesting, is it because we are passing the function from the `marker` object instead of passing a function directly (e.g. `('mouseenter', fooFunc)`)? 

Looking at [documentation](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) online, `addEventListener` should be able to take just the function object itself as a param. The code is fine as is, but I'm just curious what would cause it to execute on load instead of when the event happens.

Reply:
O, wait, I just realized I was being a dumb-dumb. My mistake. I added the brackets which ran the function instead of just passing in the function name.